src/components/graph/axis.js

```diff
@@ -24,10 +25,11 @@ class Axis extends React.Component {
       axis = axisBottom(scale);
       if (tickType === "date") {
         axis.tickFormat(timeFormat(tickFormat ||"%d %b %y"));
-        axis.ticks(ticks);
-      }
-      //TODO allow other tick type and formatting here
-    }
+        //spaced out ticks by displaying every four month
```

Comment:
This is a good call, but I think we may want to set a limit on how far back the graph displays. I'd suggest one year of history, because iirc that's what we heard from early adopters.